app/components/Settings/UserConnections/UserConnections.react.js

```diff
@@ -26,6 +26,15 @@ export default class UserConnections extends Component {
         super(props);
 		this.getStorageOnClick = this.getStorageOnClick.bind(this);
 		this.testClass = this.testClass.bind(this);
+        this.toggleSampleCredentials = this.toggleSampleCredentials.bind(this);
+
+        this.state = {showSampleCredentials: false};
+    }
+
+    
+    
+    toggleSampleCredentials() {
+        this.state.showSampleCredentials = !this.state.showSampleCredentials;
```

Comment:
this should be `setState`. `this.state` shouldn't ever be mutated.
```
this.setState({showSampleCredentials: !this.state.showSampleCredentials})
```

Calling `setState` will also trigger a re-render.